addons/dialogue_nodes/objects/DialogueParser.gd

```diff
@@ -207,6 +208,21 @@ func _process_condition(dict : Dictionary):
 	_proceed(dict[str(result).to_lower()])
 
 
+# Process the fork node data (dict).
+func _process_fork(dict : Dictionary):
+	var result = _check_fork(dict)
+	_proceed(result)
+
+
+# Checks the fork conditions based on each's option's condition, in order
+func _check_fork(dict : Dictionary):
```

Comment:
Maybe both the `_check_fork()` function and the `_process_fork()` function can just be one function.

Reply:
Removed redundant method, only `_process_fork()` remains.